salt/modules/junos.py

```diff
@@ -940,7 +947,10 @@ def install_config(path=None, **kwargs):
             finally:
                 salt.utils.files.safe_rm(template_cached_path)
 
-            if db_mode != "dynamic":
+            config_diff = None
+            if db_mode in ["dynamic", "ephemeral"]:
+                log.debug("diff is not supported for dynamic and ephemeral")
```

Comment:
Wondering if these should be 'log.warning' instead of 'log.debug', since trying an operation that is not supported needs to get reflected back to the user rather than passively ignored and only indicated if logging debug statements

Reply:
fixed via https://github.com/saltstack/salt/pull/57281/commits/87dda5eddd9593b67e1bb98caea2cb56e2bf2980